apps/src/blockly/addons/cdoBlockFlyout.ts

```diff
@@ -1,16 +1,24 @@
-import GoogleBlockly, {Block, IDraggable, Options} from 'blockly/core';
-import {FlyoutItem} from 'blockly/core/flyout_base';
-import {Svg} from 'blockly/core/utils';
+import * as GoogleBlockly from 'blockly/core';
 
 import {ExtendedWorkspaceSvg} from '../types';
 
 const svgPaths = GoogleBlockly.utils.svgPaths;
-interface CdoBlockFlyoutOptions extends Options {
+interface CdoBlockFlyoutOptions extends GoogleBlockly.Options {
   minWidth: number;
   maxWidth: number;
-  parentBlock: Block | null;
+  parentBlock: GoogleBlockly.Block | null;
+}
+
+enum FlyoutItemType {
+  BLOCK = 'block',
+  BUTTON = 'button',
 }
 
+interface FlyoutItem {
```

Comment:
It looks like `FlyoutItem` is exported: https://github.com/google/blockly/blob/60da7d87625080ee44093c45a56689456883d6f6/core/flyout_base.ts#L1372-L1376